charts/dvpe-deployment-gloo/README.md

```diff
@@ -156,77 +156,77 @@ The following table lists the configurable parameters of the chart and its defau
-| gloo.namespace | string | `"gloo-system"` | `Namespace` where all Gloo resources are deployed. |
-| gloo.upstream.namespace | string | `"gloo-system"` | `Namespace` where gloo upstream is deployed. |
-| gloo.virtualservice.spec.sslConfig.secretRef.name | string | `"gloo-public-tls"` | Name of the secret containing the certificate information for this deployment. |
-| gloo.virtualservice.spec.sslConfig.secretRef.namespace | string | `nil` | Namespace where the secret is located. If empty, gloo namespace is used. |
-| gloo.virtualservice.spec.virtualHost.domains | string | `nil` | `DNS domain name` this service will be published to. |
-| gloo.virtualservice.spec.virtualHost.routes.additionalRoutes | string | `nil` | List of route configurations for this `VirtualService`. See [gloo VirtualService Specification](https://docs.solo.io/gloo/1.1.0/introduction/concepts/#virtual-services) for details |
-| gloo.virtualservice.spec.virtualHost.routes.appPath | string | `nil` | Path to `appUrl` where the service can be accessed. Pre-defined route in `VirtualService`. |
-| gloo.virtualservice.spec.virtualHost.routes.callbackUrlPath | string | `nil` | Path to `callbackUrl` which needs to be registered at the Identity Provider. Pre-defined route in `VirtualService`. |
-| istio.destinationRule.spec.trafficPolicy.tls.mode | string | `"ISTIO_MUTUAL"` | trafficPolicy [ClientTLSSettings-TLSmode](https://istio.io/latest/docs/reference/config/networking/destination-rule/#ClientTLSSettings-TLSmode) |
-| istio.enabled | bool | `true` | Enables mtls per workload (pod) |
-| istio.peerAuthentication.spec.mtls.mode | string | `"STRICT"` | mTLS mode for istio. [PeerAuthentication-MutualTLS-Mode](https://istio.io/latest/docs/reference/config/security/peer_authentication/#PeerAuthentication-MutualTLS-Mode) |
-| service.spec.ports.http.port | string | `"80"` | The http port the service is exposed to in the cluster. |
-| service.spec.ports.http.targetPort | string | `"80"` | The http port the service listens to and to which requests will be sent. |
-| service.spec.ports.https.port | string | `"443"` | The https port the service is exposed to in the cluster. |
-| service.spec.ports.https.targetPort | string | `"443"` | The http port the service listens to and to which requests will be sent. |
-| service.spec.type | string | `"ClusterIP"` | Specify what kind of service to deploy. See [Kubernetes Service Spec](https://kubernetes.io/docs/concepts/services-networking/service/) for details |
+| Key                                                              | Type   | Default               | Description                                                                                                                                                                                                                                                    |
```

Comment:
this table is generated via helm-docs. I would suggest that you execute helm-docs locally before you checkin. that's also the reason why the commit hooks weren't approved. 
you can have a look at https://github.com/norwoodj/helm-docs for details on how to use helm-docs